sled-agent/src/bootstrap/params.rs

```diff
@@ -21,3 +25,24 @@ pub struct SledAgentRequest {
     /// Portion of the IP space to be managed by the Sled Agent.
     pub subnet: Ipv6Subnet<SLED_PREFIX>,
 }
+
+#[derive(Clone, Copy, Debug, Serialize_repr, Deserialize_repr, PartialEq)]
+#[repr(u32)]
+pub enum Version {
+    V1 = 1,
```

Comment:
I'd suggest not using an enum here with such a large repr. The problem is that if we really have many versions, each one will have to be maintained in this enum, even if our code no longer supports that version. Otherwise, it won't deserialize successfully if we are talking to an unsupported version and we won't report what version it's looking for. It's probably easier to just use a u32 or newtype. 

The other option

Reply:
I'm happy to switch this to a plain `u32`, although it looks like your comment got cut off partway through. What's the other option? :joy:

Reply:
haha, probably left over from something I started typing and thought I deleted. There are probably other options, but u32 is my suggestion :)